uspace/lib/c/include/byteorder.h

```diff
@@ -84,6 +84,32 @@
 #define ntohs(n)  uint16_t_be2host((n))
 #define ntohl(n)  uint32_t_be2host((n))
 
+#define uint8_t_be2host(n)  (n)
+#define uint8_t_le2host(n)  (n)
+#define host2uint8_t_be(n)  (n)
+#define host2uint8_t_le(n)  (n)
+#define host2uint8_t_le(n)  (n)
```

Comment:
This set of macros for `u/int8_t` is pointless. Why add it?

Reply:
Because it allows us to use them in a macro - as a uintX_t_le2host() with X as an argument. And they are not pointless, they are "casting" a value endianity, sort of code self-documentation.